src/libraries/System.Runtime.Extensions/tests/System/Convert.FromHexString.cs

```diff
@@ -102,5 +102,19 @@ public static void ZeroLength()
         {
             Assert.Same(Array.Empty<byte>(), Convert.FromHexString(string.Empty));
         }
+
+        [Fact]
+        public static void ToHexFromHexRoundtrip()
+        {
+            for (int i = 1; i < 50; i++)
+            {
+                byte[] data = System.Security.Cryptography.RandomNumberGenerator.GetBytes(i);
+                string hex = Convert.ToHexString(data);
+                Assert.Equal(data, Convert.FromHexString(hex.ToLowerInvariant()));
+                Assert.Equal(data, Convert.FromHexString(hex.ToUpperInvariant()));
```

Comment:
Is a mixture of casing worth testing?

Reply:
It won't hurt, added test cases for mixed casing